e2e/kube/storage/examples_storage_test.go

```diff
@@ -9,18 +11,29 @@ import (
 
 var _ = Describe("Examples", func() {
 
-	Describe("when examples are specified in the docs", func() {
+	Describe("when storage related examples are specified in the docs", func() {
 
+		var (
+			kubectlHelper *testing.Kubectl
+		)
+		kubectlHelper = testing.NewKubectl()
```

Comment:
This should go into a `BeforeEach` block.

Reply:
Done.